client/components/domains/map-domain-step/index.jsx

```diff
@@ -59,8 +60,7 @@ const MapDomainStep = React.createClass( {
 
 	render: function() {
 		const suggestion = { cost: this.props.products.domain_map.cost_display, product_slug: this.props.products.domain_map.product_slug },
```

Comment:
OK, since I don't like ambiguity and depending on how the application behaves _now_, I'll add a safeguard for this 👍